catalog/transformer/catalog-transformer-common/src/main/java/ddf/catalog/transformer/common/tika/MetacardCreator.java

```diff
@@ -153,9 +192,70 @@ public static Metacard createMetacard(
     setAttribute(
         metacard, Mp4MetacardType.AUDIO_SAMPLE_RATE, metadata.get(XMPDM.AUDIO_SAMPLE_RATE));
 
+    applyAlternateMappings(metadata, metacard);
+
     return metacard;
   }
 
+  private static void applyAlternateMappings(final Metadata metadata, Metacard metacard) {
+    MetacardType metacardType = metacard.getMetacardType();
+    for (Map.Entry<String, List<String>> entry : ALTERNATE_METADATA_KEY_MAPPING.entrySet()) {
+      String attributeName = entry.getKey();
+      if (StringUtils.isBlank(attributeName)) {
+        continue;
+      }
+
+      Attribute attribute = metacard.getAttribute(attributeName);
+      if (attribute == null) {
+        for (String mapping : entry.getValue()) {
+          String[] values = metadata.getValues(mapping);
+          List<Serializable> attrValues = new ArrayList<>();
+          for (String value : values) {
+            Serializable result = getMetadataValue(attributeName, value, metacardType);
+            if (result != null) {
+              attrValues.add(result);
+            }
+          }
+          if (!attrValues.isEmpty()) {
+            metacard.setAttribute(new AttributeImpl(attributeName, attrValues));
+          }
+        }
+      }
+    }
+  }
+
+  private static Serializable getMetadataValue(
+      String attributeName, String value, MetacardType metacardType) {
+    Serializable result = null;
+    if (StringUtils.isNotBlank(value)) {
+      AttributeDescriptor descriptor = metacardType.getAttributeDescriptor(attributeName);
+
+      if (descriptor != null) {
+        AttributeFormat attributeFormat = descriptor.getType().getAttributeFormat();
+
+        if (attributeFormat == AttributeFormat.INTEGER) {
+          try {
+            result = Integer.valueOf(value.trim());
+          } catch (NumberFormatException nfe) {
+            LOGGER.debug(
+                "Expected integer but was not integer. This is expected behavior when "
+                    + "a defined integer attribute does not exist on the ingested product.");
+          }
+        } else if (attributeFormat == AttributeFormat.DOUBLE) {
+          result = getDoubleFromString(value);
+        } else if (attributeFormat == AttributeFormat.DATE) {
+          result = convertDate(value);
+        } else {
+          result = value;
```

Comment:
❓ There are other non-String `AttributeFormat` types. Is there a reason we do not support parsing those values from the String value?

https://github.com/codice/ddf/blob/master/platform/solr/solr-query/src/main/java/org/codice/solr/query/AttributeFormat.java

Reply:
No reason other than I didn't need them right now.  I can look at adding them though.  Probably a good idea since it is configurable now.

Reply:
I added most of the others.  I didn't think Geometry, XML, and Object made sense to support here, though.  Geometry would probably be the one most in question, but given how it works elsewhere in this code (creating geometries from multiple values), it would probably require a decent amount of changes to support directly rather having a simple attribute mapping.

Reply:
I think this is enough for now until we have a geo use case we actually need to support.